r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -317,6 +319,9 @@ test_that("median()", {
     tbl,
     warning = "median\\(\\) currently returns an approximate median in Arrow"
   )
+  # silence warnings
+  rlang::local_options(rlib_warning_verbosity = "quiet")
+
```

Comment:
Hmm, this will quiet all rlang-type warnings, which is probably not quite what we want. Maybe this should be `"default"` instead of `"quiet"`? Though, this is moot if we go in the other direction up above.

Reply:
I tried `default`, but in some cases (maybe consistent? but I don't want to promise that) the `.frequency = "once"` ones still threw a warning afterwards. Easier/safer to just make it verbose at the start of the file and back to default at the end; let me try it out

Reply:
🤔 I wonder if changing that setting actually resets the counter? 

Anyway, for our purposes, during tests, we probably do want it to always end up firing cause that's easier to catch + test or ignore.

Reply:
Changed and pushed